### PR TITLE
[FEATURE] Host meta endpoint

### PR DESCRIPTION
**Pull request by stevenzeiler**
[FEATURE] Redirect root to host meta, link to docs
[FEATURE] Display codius host version in host meta
[FEATURE] List billing modules supported and prices
[FEATURE] Display RSA public key of host
[Finishes #89446660]
[Finishes #89445636]

### CODE REVIEW

**Comment by stevenzeiler** [comment]
+r

File: test/http/host_meta.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+var Promise   = require('bluebird')
+var codius    = require('../../')
+var assert    = require('assert')
+var supertest = require('supertest')
+var server    = require('../../lib/application')(codius)
+var http      = supertest(server)
+var fs        = require('fs')
+var path      = require('path')
+
+describe('Billing Service HTTP Interface', function() {
```
Comment (wilsonianb) [inline test/http/host_meta.js]: Fix name

File: test/http/host_meta.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+var fs        = require('fs')
+var path      = require('path')
+
+describe('Host Meta and Root Endpoints', function() {
+  var version, publicKey
+
+  before(function(done) {
+    fs.readFile(path.join(__dirname+'/../../package.json'), function(error, file) {
+      version = JSON.parse(file.toString()).version
+      publicKey = fs.readFileSync(__dirname+'/../../server.crt').toString()
```
Comment (wilsonianb) [inline test/http/host_meta.js]: It looks like this path doesn't work with circleci, which does this:
`yes US | openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/contracts/server.key -out /tmp/contracts/server.crt`
then
`SSL_KEY=/tmp/contracts/server.key SSL_CERT=/tmp/contracts/server.crt CONTRACTS_STORAGE=/tmp/contracts/ npm test`

File: test/http/host_meta.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+var uuid      = require('uuid')
+var Promise   = require('bluebird')
```
Comment (wilsonianb) [inline test/http/host_meta.js]: `uuid` and `Promise` aren't used

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**